HTTP_200/templates/bulkuser.html

```diff
@@ -0,0 +1,11 @@
+<html>
+<head>
+	<title> BulkUser</title>
+</head>
+<body>
+	<p>All the users have been created.</p>
+	<p><a href="/admin/"> Return to admin pannel again </a></p> 
```

Comment:
@RishabhJain2018 never give direct url links. Use `{% url ``%}`

Anyways, I am merging this time. Keep this in mind from next time. :smile: